students/jsward/session01/breakme.py

```diff
@@ -0,0 +1,7 @@
+# Break things
+
+def name_error():
+    #print("blah{}".format(undefined_variable))
+    pass
+
```

Comment:
This is a great start! Don't forget to add the other errors: TypeError, AttributeError, SyntaxError